src/main/java/com/thirdcc/webapp/web/rest/DebtResource.java

```diff
@@ -40,6 +43,25 @@
     public DebtResource(DebtService debtService) {
         this.debtService = debtService;
     }
+    
+    /**
+     * {@code POST  /debts} : Create a new debt.
+     *
+     * @param debtDTO the debtDTO to create.
+     * @return the {@link ResponseEntity} with status {@code 201 (Created)} and with body the new debtDTO, or with status {@code 400 (Bad Request)} if the debt has already an ID.
+     * @throws URISyntaxException if the Location URI syntax is incorrect.
+     */
+    @PostMapping("/debts")
+    public ResponseEntity<DebtDTO> createDebt(@RequestBody DebtDTO debtDTO) throws URISyntaxException {
```

Comment:
@CarlosSia 
Can include Authority?
Any test written for this API?

Reply:
@HowardYaw 
Does this method need any Authority? Because if the user register for an event they should be the one who create the debt right?

No test for this API, will create it

Reply:
I think, should have authority. Only crew/admins can create debt

For create debt during user register event, should be:
- Inject debt DAO to register service and create